jsonrpc/bridge_endpoint.go

```diff
@@ -0,0 +1,20 @@
+package jsonrpc
+
+import (
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+// bridgeStore interface provides access to the methods needed by bridge endpoint
+type bridgeStore interface {
+	GenerateExitProof(exitID, epoch, checkpointBlock uint64) ([]types.Hash, error)
+}
+
+// Bridge is the bridge jsonrpc endpoint
+type Bridge struct {
+	store bridgeStore
```

Comment:
General question, I see that it's the pattern but why is it called `store`?

Reply:
No idea 😄 . I just went along with rest of the endpoints

Reply:
Most likely it was me who called it `store`.

Reply:
I would call it `bridgeDataProvider` or something like that. Also Goran put it GenerateExitProof to the Consensus inteface, but my gut feeling is that it doesn't belong there...

Reply:
> Most likely it was me who called it `store`.

@ferranbt I know you like words such as `backend`, `state`, `snapshot`, but `store` is new one... haha 😄 Just kidding a little bit. 

Anyhow I'm aware that occasionally it is hard to come up with some meaningful name, but on the other hand it also can be hard to figure out which module is in charge for which part when we have repeating names here and there.